lib/tasks/orangeindex.rake

```diff
@@ -167,6 +167,16 @@ namespace :liberate do
     solr.commit
   end
 
+  desc "Index a single MARC XML file against SET_URL"
+  task :index_file, [:file_name] => :environment do |_t, args|
+    solr_url = ENV['SET_URL'] || default_solr_url
+    file_name = args[:file_name]
```

Comment:
pul codebases tend to use environment variables instead of arguments with rake tasks. I think we should use `ENV['FILE']` or similar.

Reply:
Thanks for the pointer. Done.